VERSION

```diff
@@ -1 +1 @@
-1.0.0
\ No newline at end of file
+2.0.3
```

Comment:
this VERSION file can be removed because we inject version number during the build time